docs/guide/examples.rst

```diff
@@ -209,9 +209,11 @@ These dictionaries are randomly initialized on the creation of the environment a
   model.learn(total_timesteps=100_000)
 
 
-Using Callback: Monitoring Training
------------------------------------
+Using Callbacks
```

Comment:
Please keep the same heading level, 
I would rename the two section to "Callbacks: Monitoring Training" and "Callbacks: Evaluate Agent Performance"